azure-common-auth/pom.xml

```diff
@@ -48,8 +49,8 @@
 
   <dependencies>
     <dependency>
-      <groupId>com.microsoft.azure.v3</groupId>
-      <artifactId>azure-client-runtime</artifactId>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-common</artifactId>
       <version>3.0.0-SNAPSHOT</version>
```

Comment:
We should consider rolling back the versions to 1.0.0 prior to release. Thoughts?

Reply:
sure, 3.x.x is not valid any more. May be we can start with `0.1.0` to indicate that it's in preview?

Reply:
Going with 1.0.0-SNAPSHOT as discussed below.